recipes/qt-multimedia/conda_build_config.yaml

```
@@ -0,0 +1,5 @@
+# https://doc.qt.io/qt-6/macos.html
+c_stdlib_version:          # [osx]
+  - '11.0'                 # [osx]
+MACOSX_SDK_VERSION:        # [osx]
+  - '12.3'                 # [osx]
```

There's a gstreamer migration currently, and qt-main has already moved on, so if you want the build to work right away, you can add the pins from the [migrator](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/gstreamer124.yaml) here
```suggestion
MACOSX_SDK_VERSION:        # [osx]
  - '12.3'                 # [osx]

gstreamer:
- '1.24'
gst_plugins_base:
- '1.24'
```